src/main/app/claims/claimModelConverter.ts

```diff
@@ -36,6 +39,8 @@ export class ClaimModelConverter {
     draftClaim.claimant['type'] = 'individual'
 
     draftClaim.claimant.mobilePhone = draftClaim.claimant.mobilePhone.number as any
+
+    draftClaim.claimant.dateOfBirth = draftClaim.claimant.dateOfBirth.date.asString() as any
```

Comment:
don't we always want to keep dates as Moment?

Reply:
this function converts to moment and then formats it how as want it, asString() may not be the best name, but it is what we want to do here
```
this.toMoment().format(DATE_FORMAT)
```